<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPropertiesPanel</class>
 <widget class="QWidget" name="DataPropertiesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="FileName">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="ActiveScalars"/>
   </item>
   <item>
    <widget class="QLabel" name="DataRange">
     <property name="text">
      <string>DataRange</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="NumVoxels">
     <property name="text">
      <string>NumVoxels</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="MemSize">
     <property name="text">
      <string>MemSize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="ScalarsTable"/>
   </item>
   <item>
    <widget class="QWidget" name="PropertiesButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="LengthWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="leftBracketLabel">
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="rightBracketLabel">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="zLabel">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="multiplyLabel2">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="xLengthBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="zLengthBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="multiplyLabel1">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="yLengthBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLineEdit" name="unitBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="7" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="unitLabel">
          <property name="text">
           <string>Units</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="SetTiltAnglesButton">
     <property name="text">
      <string>Set Tilt Angles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="TiltAnglesTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FileName</tabstop>
  <tabstop>xLengthBox</tabstop>
  <tabstop>yLengthBox</tabstop>
  <tabstop>zLengthBox</tabstop>
  <tabstop>unitBox</tabstop>
  <tabstop>SetTiltAnglesButton</tabstop>
  <tabstop>TiltAnglesTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
